### ISSUE TITLE
Binaries for linux arm64

### ISSUE THREAD

**Issue by tsimoshka** [samuelcolvin/watchfiles#120]
Hi,

It would be nice to have pre-build binaries for ARM Linux on PyPI.

The thing is that by default docker for mac on Apple Silicon macs runs containers and image builds for linux/arm64 platform and to install the package one needs to either install a rust compilator in container/image or run containers/builds under linux/amd64 emulation, which is significantly slower than the default platform.

**Comment by samuelcolvin** [comment]
Would that be covered by the `manylinux-aarch64` case [here](https://cibuildwheel.readthedocs.io/en/stable/#what-does-it-do)?

Do you need `musllinux-aarch64` case too? I guess it might be useful for alpine

**Comment by tsimoshka** [comment]
I suppose so, but not really sure. I will try to build it locally tomorrow and check if it works on python base images including alpine. Will report back.

**Comment by samuelcolvin** [comment]
See #121 - you can download the artifacts and install those wheels directly.

**Comment by tsimoshka** [comment]
Thanks for the quick fix!

It works ok. Just tested with the following Dockerfile:

```dockerfile
FROM --platform=linux/amd64 python:3.10 as pre_ok
RUN mkdir /pypi
COPY pypi_files_pre/ /pypi/
RUN pip install --no-index --no-deps --find-links=/pypi watchfiles

FROM --platform=linux/arm64 python:3.10 as pre_fail
RUN mkdir /pypi
COPY pypi_files_pre/ /pypi/
RUN pip install --no-index --no-deps --find-links=/pypi watchfiles

FROM --platform=linux/arm64 python:3.10 as post_ok
RUN mkdir /pypi
COPY pypi_files_post/ /pypi/
RUN pip install --no-index --no-deps --find-links=/pypi watchfiles

FROM --platform=linux/arm64 python:3.10-alpine as post_musl_ok
RUN mkdir /pypi
COPY pypi_files_post/ /pypi/
RUN pip install --no-index --no-deps --find-links=/pypi watchfiles
```

pypi_files_pre/ is a folder with artifacts from the https://github.com/samuelcolvin/watchfiles/actions/runs/2116922088
pypi_files_post/ is a folder with artifacts from the https://github.com/samuelcolvin/watchfiles/actions/runs/2116389382

```bash
docker build --target pre_ok . # regular build on linux/amd64, works ok
docker build --target pre_fail . # linux/arm64 build, fails as expected
# both work ok with the new aritfacts
docker build --target post_ok .
docker build --target post_musl_ok .
```

**Comment by samuelcolvin** [comment]
fixed by #121. Builds are now really slow, but those binaries are included.
